fe/model_rabfe.py

```diff
@@ -233,14 +242,15 @@ def _predict_a_to_b(
             self.host_system
         )
 
-        k_core = 75.0
-        core_params = np.zeros_like(core_idxs).astype(np.float64)
+        k_core = 30.0
```

Comment:
If this value of `k_core` is only expected to be appropriate for a limited range of `len(combined_core_idxs)`, could we add a warning when `len(combined_core_idxs)` is outside of that range?

Reply:
I think this will require more validation. I don't have a good sense for this right now